prep-javascript-objects/main.js

```diff
@@ -0,0 +1,19 @@
+const person = {
+  firstName: 'Chris',
+  lastName: 'Mendoza',
+  hobby: 'Playing Video Games',
+  job: 'Accountant',
+  previousJob: 'Billing Clerk'
+};
```

Comment:
You can add new properties to an object later in a program using dot notation.

```js
person.job = 'Accountant';
```

Reply:
Here's a new screenshot of my updated console
![prep-javascript-objects 2](https://user-images.githubusercontent.com/90743100/150043230-3763c136-00a8-4d08-bf24-40540df545f3.png)